pangraph.cabal

```diff
@@ -18,15 +18,15 @@ tested-with:
 library
   hs-source-dirs:      src
   exposed-modules:     Pangraph
-  other-modules:       Parsers.GmlP
-                     , Parsers.Discrim
+  other-modules:       Parsers.GmlParser
```

Comment:
While we are renaming things, I'd recommend using a full name for this parser: `GraphMLParser`. I find `Gml` a bit too cryptic.